tutorial/aqua/quick_start.md

```diff
@@ -0,0 +1,186 @@
+
+```bash
+fldist --node-id 12D3KooWKnEqMfYo9zvfHmqTLpLdiHXPe4SVqUWcWHDJdFGrSmcA  get_modules --pretty| grep -B 13 -A 1  greeting
+  {
+    "config": {
+      "logger_enabled": true,
+      "logging_mask": null,
+      "mem_pages_count": 100,
+      "mounted_binaries": null,
+      "wasi": {
+        "envs": null,
+        "mapped_dirs": null,
+        "preopened_files": []
+      }
+    },
+    "hash": "9af450928626beefab6c055bd6d6ddb4ae9c59344614cee7083e7be7d5fc93c2",
+    "name": "greeting"
+  },
+root@56892f4726bb:/workspaces/devcontainer/aqua-tutorial#
+```
+
+Now that we know our service exists, we can check on its interface(s) also with the`flidst` commandline tool:
+
+```bash
+root@56892f4726bb:/workspaces/devcontainer/aqua-tutorial# fldist --node-id 12D3KooWKnEqMfYo9zvfHmqTLpLdiHXPe4SVqUWcWHDJdFGrSmcA  get_interface --id c9a315de-4fe2-4730-8f40-9209428383bc --expand
+{
+  "function_signatures": [
+    {
+      "arguments": [
+        [
+          "name",
+          "String"
+        ],
+        [
+          "greeter",
+          "Boolean"
+        ]
+      ],
+      "name": "greeting",
+      "output_types": [
+        "String"
+      ]
+    }
+  ],
+  "record_types": []
+}
+root@56892f4726bb:/workspaces/devcontainer/aqua-tutorial#
+```
+
+That is, our service takes a _String_ parameter and a _boolean_ parameter to either return "Hi %name%" or "Bye %name%". With that, we have all the information needed to use our distributed service. 
+
+Create a new directory *aqua-tutorial* in your workspace and add am *aqua* dir so that you have a structure like about so:
+
+```bash
+root@56892f4726bb:/workspaces/devcontainer/aqua-tutorial# ls -lrth
+total 4.0K
+drwxr-xr-x 2 root root 4.0K Jun  1 23:59 aqua
+root@56892f4726bb:/workspaces/devcontainer/aqua-tutorial#
+```
+
+In the *aqua* dir, add the following Aqua script *greeter.aqua*:
+
+```aqua
+-- greeter.aqua                             <-- 1
+service Local("returnService"):             <-- 2
+  run: string -> ()
+
+service Greeting("service-id"):             <-- 3
+    greeting: string, bool -> string
+
+func greeter(name: string, greet: bool, node: string, service_id: string):  <-- 4
+    on node:                                                      
+      Greeting service_id
+      res <- Greeting.greeting(name, greet)
+    Local.run(res)
+```
+
+Basically, we just created a script that instructs the Aqua runtime (Aqua VM) on node *peer_id* to execute the service *service_id* with the parameters *name* and *greet* and then to return the result to the local application. This is all happening in the *greeter* func (4) and uses the service interface representation of the deployed service, *service Greeting* (3) and the *service Local* (2) to accomplish our goal. Note, inline comments in Aqua are prefixed by a double dash *--* (1).
+
+Now that we have or script, let's compile it to AIR. In your *aqua-tutorial* directory, create a dir *air-scripts* and call the aqua compiler:
```

Comment:
```suggestion
Now that we have our script, let's compile it to AIR. In your *aqua-tutorial* directory, create a dir *air-scripts* and call the aqua compiler:
```

Reply:
i feel like removing AIR output from the tutorial. maybe tell interested users to have a gander at the file, if so inclined ?

Reply:
Yes, I think that's a good idea. I don't see a need to show AIR.